src/ServicePulse.Host/vue/src/components/monitoring/graphLines.ts

```diff
@@ -12,19 +13,20 @@ export function useGraph(plotdata, minimumyaxis) {
     const values = (() => {
       let result = plotData.points;
       if (result.length === 0) {
-        result = new Array(10).fill(0);
+        result = new Array(toValue(minPoints)).fill(0);
       }
       return result;
     })();
     const xTick = 100 / (values.length - 1);
-    const coordinates = values.reduce((points, yValue, i) => [...points, [i * xTick, yValue]], []);
+    const coordinates = values.reduce((points: Coordinate[], yValue, i) => [...points, [i * xTick, yValue] as Coordinate], []);
     valuesPath.value = new Path().startAt(coordinates[0]).followCoordinates(coordinates.slice(1)).toString();
     valuesArea.value = new Path().startAt([0, 0]).followCoordinates(coordinates).lineTo([100, 0]).close().toString();
 
     average.value = plotData.average;
     //TODO: why is this called minimumYaxis when it's only used to determine the maxYaxis?
     // should the graph actually set the min y value rather than leave it at 0?
```

Comment:
If all of your values are between 2 and 3, then the maxYaxis will be set to 10, so this is the _minimum_ that the max Y axis can be.

If all of your values are between 11 and 12, then the maxYaxis will be set to 12, which is above the minimum Y axis of 10.